api/catalog/settings.py

```diff
@@ -113,7 +113,8 @@
     "SCOPES": {
         "read": "Read scope",
         "write": "Write scope",
-    }
+    },
+    "ACCESS_TOKEN_EXPIRE_SECONDS": 3600 * 12,
```

Comment:
Thanks for your contribution @raghuvar-arora-au2! The updated time looks good to me according to the [OAuth Toolkit docs](https://django-oauth-toolkit.readthedocs.io/en/1.7.1/settings.html?highlight=ACCESS_TOKEN_EXPIRE_SECONDS#access-token-expire-seconds). It would be nice to make this configurable with environment variables too.
```suggestion
    "ACCESS_TOKEN_EXPIRE_SECONDS": config(
        "ACCESS_TOKEN_EXPIRE_SECONDS", default=3600 * 12, cast=int
    ),
```